resolwe/flow/models/process.py

```diff
@@ -156,3 +157,35 @@ class Meta(BaseModel.Meta):
     """
     process scheduling class
     """
+
+    def get_resource_limits(self):
+        """Get the core count and memory usage limits for this process.
+
+        :return: A dictionary with the resource limits, containing the
+            following keys:
+
+            - ``memory``: Memory usage limit, in MB.
+            - ``cores``: Optional. Core count limit.
+
+        :rtype: dict
+        """
+        # Get limit defaults and overrides.
+        limit_defaults = settings.get('FLOW_DOCKER_LIMIT_DEFAULTS', {})
+        limit_overrides = settings.get('FLOW_DOCKER_LIMIT_OVERRIDES', {})
+
+        limits = {}
+
+        resources = self.requirements.get('executor', {}).get('resources', {})
+
+        if 'cores' in resources:
+            limits['cores'] = int(resources['cores'])
```

Comment:
@kostko, @tjanez would it ok to set cores requirement to 1 by default?

Reply:
I think it would be ok.